monorepo/packages/wallet-connectors/src/NearConnector/index.ts

```diff
@@ -131,3 +138,44 @@ export class NearConnector extends AbstractConnector {
     return false;
   }
 }
+
+function getNearMainnetConfig() {
```

Comment:
`getNearMainnetConfig`, `getNearConfig` is already defined in `near.ts`

Reply:
same of above